### ISSUE TITLE
Why is fakeredis being sorted as a first-party import?

### ISSUE THREAD

**Issue by BenQuigley** [timothycrosley/isort#1165]
This might just be user error of some kind that I'm not competent enough to understand, but why is [fakeredis](https://pypi.org/project/fakeredis/) being classified as a first-party import rather than third-party? For example:
```
$ vi example.py
  1 import json
  2 
  3 import fakeredis
  4 import requests
  5 
  6 import bar
  7 
  8 print("Hello")
$ python3 -m isort example.py
$ vi example.py
  1 import json
  2
  3 import requests
  4
  5 import bar
  6 import fakeredis
  7
  8 print("Hello")
```

My isort version is 4.3.21 on OS X.

**Comment by Siecje** [comment]
It's because fakeredis is not installed in the Python environment you are using to run isort. I believe this has been fixed in the unreleased `develop` branch.

**Comment by BenQuigley** [comment]
Thank you!

**Comment by timothycrosley** [comment]
These type of issues should be fixed in the latest 5.0.0 release of isort: https://timothycrosley.github.io/isort/docs/major_releases/introducing_isort_5/

Thanks for everyone for reporting them to help me prioritize!

~Timothy
